doc/releases/changelog-dev.md

```diff
@@ -192,7 +192,7 @@
   [(#6632)](https://github.com/PennyLaneAI/pennylane/pull/6632)
   [(#6653)](https://github.com/PennyLaneAI/pennylane/pull/6653)
 
-<h4>New API for Default mixed</h4>
+<h4>New API for Default Mixed</h4>
```

Comment:
```suggestion
<h4>New device API for default.mixed</h4>
```

Reply:
@isaacdevlugt the standard in the changelogs looks to be "Default Mixed", "Default Tensor", "Default Qubit", etc, when the device name is in the header.

Reply:
You're right 🤔 ... interesting... Maybe in a proper header/title it can be "Default Mixed" but in the body it should be `default.mixed` (the user-facing short name).